src/Glpi/Marketplace/View.php

```diff
@@ -912,8 +806,8 @@ public static function getButtons(string $plugin_key = ""): string
                                        ' . _x("button", "Uninstall") . '
                                    </a>',
                 'content' => sprintf(
-                    __s('By uninstalling the "%s" plugin you will lose all the data of the plugin.'),
```

Comment:
This sounds weird we have to change a source string. Should not `__s()` behave the same than `htmlescape()`?

Reply:
Also `__s` is indeed just wrapping the translated string in a `htmlspecialchars` call with the default flags.

Reply:
`__s` should not be used here, `content` is automatically escaped by the template. I fixed this.